src/robusta/integrations/slack/sender.py

```
@@ -12,12 +12,14 @@
 from slack_sdk import WebClient
 from slack_sdk.errors import SlackApiError
 from slack_sdk.http_retry import all_builtin_retry_handlers
+from slack_sdk.http_retry.builtin_handlers import RateLimitErrorRetryHandler
 from robusta.core.sinks.slack.templates.template_loader import template_loader
 
 from robusta.core.model.env_vars import (
```

Flake8 (7.3.0)

[error] 'robusta.core.model.env_vars.HOLMES_ASK_SLACK_BUTTON_ENABLED' imported but unused

(F401)